pyspark/python_package/python_submit_yarn_cluster.sh.template

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This template is to show how to submit python job with dependencies on spark yarn clusters.
+
+BigDL_HOME=/root/BigDL
```

Comment:
I think we can leave the value as ... for BigDL_HOME, SPARK_HOME in the template